pyvisa_py/tcpip.py

```diff
@@ -234,10 +235,11 @@ def __init__(
             rpc.RawTCPClient.__init__(self, host, prog, vers, port, open_timeout)
 
 
-@Session.register(constants.InterfaceType.tcpip, "VXI11")
 class TCPIPInstrVxi11(Session):
     """A TCPIP Session built on socket standard library using VXI-11 protocol."""
 
+    session_type = (constants.InterfaceType.tcpip, "VXI11")
```

Comment:
Same here VISA only provide one attribute or so to discriminate between VXI11 and HiSLIP.

Reply:
I thought about this, tried your suggestion, and I agree that using "INSTR" for the resource class is the correct thing to do here.